tests/e2e/utils/activate-amp-and-set-mode.js

```diff
@@ -65,7 +65,7 @@ export const activateAMPWithMode = async (
 			},
 		] );
 	} else {
-		// eslint-disable-next-line react-hooks/rules-of-hooks
+		// If no sharedRequestInterception is passed, we need to create a new request interception.
```

Comment:
In the previous review when I mentioned this part not being clear as to why it's needed I was more speaking about the added request interception as a whole rather than just this part.

Reply:
Thanks, @aaemnnosttv